Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```
@@ -637,3 +640,61 @@
         role: JobBorg
         time: 3600 #1 hour
     - type: GhostTakeoverAvailable
+
+- type: entity
+  name: onestar mecha
+  id: Onestar
+  parent: PlayerSiliconBase
+  components:
+  - type: Drone
+    tools:
+      - id: Minigun
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)